Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,43 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            //Tell me if I did the task wrongly, because I wasn't really sure if I had had to define them and not ask them in.
+            //Read name, surename, age, weight (in kg) and height (in cm) from console
+            // calculate BMI index
+            string namePerson1 = "Tom";
+            string namePerson2 = "Daniel";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add the 'const' modifier to 'namePerson2'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4846233)